cmd/organization.go

```diff
@@ -0,0 +1,238 @@
+
+import (
+	"context"
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/uselagoon/lagoon-cli/pkg/output"
+	l "github.com/uselagoon/machinery/api/lagoon"
+	lclient "github.com/uselagoon/machinery/api/lagoon/client"
+	s "github.com/uselagoon/machinery/api/schema"
+)
+
+var addOrgCmd = &cobra.Command{
+	Use:     "organization",
+	Aliases: []string{"o"},
+	Short:   "Add a new organization to Lagoon",
+	PreRunE: func(_ *cobra.Command, _ []string) error {
+		return validateTokenE(lagoonCLIConfig.Current)
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		debug, err := cmd.Flags().GetBool("debug")
+		if err != nil {
+			return err
+		}
+		organizationName, err := cmd.Flags().GetString("organization")
+		requiredInputCheck("Organization name", organizationName)
+		if err != nil {
+			return err
+		}
+		organizationFriendlyName, err := cmd.Flags().GetString("friendlyName")
+		if err != nil {
+			return err
+		}
+		organizationDescription, err := cmd.Flags().GetString("description")
+		if err != nil {
+			return err
+		}
+		organizationQuotaProject, err := cmd.Flags().GetUint("quotaProject")
+		if err != nil {
+			return err
+		}
+		organizationQuotaGroup, err := cmd.Flags().GetUint("quotaGroup")
+		if err != nil {
+			return err
+		}
+		organizationQuotaNotification, err := cmd.Flags().GetUint("quotaNotification")
+		if err != nil {
+			return err
+		}
+		organizationQuotaEnvironment, err := cmd.Flags().GetUint("quotaEnvironment")
+		if err != nil {
+			return err
+		}
+		organizationQuotaRoute, err := cmd.Flags().GetUint("quotaRoute")
+		if err != nil {
+			return err
+		}
+
+		current := lagoonCLIConfig.Current
+		token := lagoonCLIConfig.Lagoons[current].Token
+		lc := lclient.New(
+			lagoonCLIConfig.Lagoons[current].GraphQL,
+			lagoonCLIVersion,
+			&token,
+			debug)
+
+		organizationInput := s.AddOrganizationInput{
+			Name:              organizationName,
+			FriendlyName:      organizationFriendlyName,
+			Description:       organizationDescription,
+			QuotaProject:      organizationQuotaProject,
+			QuotaGroup:        organizationQuotaGroup,
+			QuotaNotification: organizationQuotaNotification,
+			QuotaEnvironment:  organizationQuotaEnvironment,
+			QuotaRoute:        organizationQuotaRoute,
+		}
+		org := s.Organization{}
+		err = lc.AddOrganization(context.TODO(), &organizationInput, &org)
+		handleError(err)
+
+		resultData := output.Result{
+			Result: "success",
+			ResultData: map[string]interface{}{
+				"Organization Name": organizationName,
+			},
+		}
+		output.RenderResult(resultData, outputOptions)
+		return nil
+	},
+}
+
+var deleteOrgCmd = &cobra.Command{
+	Use:     "organization",
+	Aliases: []string{"o"},
+	Short:   "Delete an organization",
+	PreRunE: func(_ *cobra.Command, _ []string) error {
+		return validateTokenE(lagoonCLIConfig.Current)
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		debug, err := cmd.Flags().GetBool("debug")
+		organizationName, err := cmd.Flags().GetString("organization")
```

Comment:
Same as the other one, change it to `name`, and any other occurrences, looks like there are others